app/controllers/settings/password.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  actions: {
+    changePassword(passwordData) {
+      this.set('isLoading', true);
+      let payload = {
+        'data': {
+          'old-password' : passwordData.passwordCurrent,
+          'new-password' : passwordData.passwordNew
+        }
+      };
+      this.get('loader')
+        .post('/auth/change-password', payload)
+        .then(data => {
+          if (data.error) {
```

Comment:
@niranjan94 The server is sending response every time. When the password is incorrect, it sends:
```
{
"error": "Wrong password"
}
```
So I guess it wont be coming in catch. I tried that in first place but then realised that the server always returns response, its just an object( containing user's info ) when its success else it contains the error mentioned above.